docker/README.md

```diff
@@ -16,7 +16,7 @@ Linux users (including those using a VM), can install Docker Engine or Docker De
 
 ### A note on `docker-compose` and `docker compose`
 
-As of early 2023, following the installation instructions on Docker's website will install either Docker Desktop, which includes Docker Compose v2, or `docker-ce` and `docker-compose-plugin` (Linux only), both of which obviate the need to install `docker-compose` v1 separately.
+As of early 2023, following the installation instructions on Docker's website will install either Docker Desktop, which includes Docker Compose v2, or `docker-ce` and `docker compose plugin` (Linux only), both of which obviate the need to install `docker-compose` v1 separately.
```

Comment:
```suggestion
As of early 2023, following the installation instructions on Docker's website will install either Docker Desktop, which includes Docker Compose v2, or `docker-ce` and `docker-compose-plugin` (Linux only), both of which obviate the need to install `docker-compose` v1 separately.
```
Let's keep `docker-compose-plugin` here as it's the name of the package: https://docs.docker.com/compose/install/linux/#install-using-the-repository